common/beerocks/bwl/CMakeLists.txt

```diff
@@ -10,7 +10,8 @@ file(GLOB_RECURSE bwl_common_sources ${MODULE_PATH}/common/*.c*)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
 
 # Default defaults
-set(BWL_TYPE "DUMMY")
+set(BWL_TYPE "DUMMY" CACHE STRING "BWL type")
```

Comment:
It is already defined as a cache variable below. So that one should be removed. 

However, I still don't think it's OK then. IIUC a plain variable overrides a cache variable, and both can be defined at the same time. So the `set` commands within the `TARGET_PLATFORM` condition below will override whatever was set in the cache variable. I can't say that CMake is entirely clear for me though, so I could be wrong about this.

Even so, I think it would be much clearer to do what I suggested before: change the variable name to `BWL_TYPE_DEFAULT` and use that in the definition of the cache variable below.